### PR TITLE
Register page

### LINKED ISSUE #8: Create Register Page
Create basic register page to get user info.
Info we need:
First name
Last name
email
major
year

### LINKED ISSUE #17: Familiarize with Pyrbase API

### PR DESCRIPTION
**Pull request by MantaRey**
Register.html (Form Page)
RegisterAlt.html (not important)
App.py route added

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -28,7 +28,24 @@ def update():
...
+@app.route("/register", methods=('GET','POST'))
+def register_page():
+    print("Can You See This")
+    if request.method == 'POST':
+        firstName = request.form['firstName']
+        lastName = request.form['lastName']
+        email = request.form['email']
+        major = request.form['major']
+        year = request.form['year']
+        username = firstName + lastName
```
Comment (josueal1) [inline app.py:45]: @guilleeh @MantaRey @jaime-leal  we might want to pass these variables into a pyrebase function. we can write the function ourselves

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,6 @@ build/
 
 # The . idea folder (hidden on OS X) in the solution root contains IntelliJ's project specific settings files.
 .idea/
+
+# Nothing in here for now
+static/
```
Comment (josueal1) [inline .gitignore]: im not sure if we want to keep the `static/` folder in .gitignore bc we might add custom css files and might want to track those changes

**Review by josueal1 — APPROVED** [review]
Look good to me (LGTM) !  Lmk your ideas on my comments tho

File: app.py (hunk 1 above)
Comment (MantaRey) [reply app.py:45]: Yes! Definitely we will need to do that. I was just seeing if I was able to read from the form and temporarily "store" the data.

File: .gitignore (hunk 2 above)
Comment (MantaRey) [reply .gitignore]: Yeah, I just wasn't sure how you guys were going to go about CSS so i put it in .gitignore for now, but I can change that right now!

Comment (guilleeh) [reply .gitignore]: Yeah can you take out static from gitignore? We're gonna need that folder

**Review by guilleeh — APPROVED** [review]
LGTM, remove static from .gitignore

**Pull request merged**